sql-blueprinting/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "sql-blueprinting",
+  "version": "0.1.0",
+  "description": "",
+  "main": "index.js",
+  "author": "FotisVasilopoulos",
+  "scripts": {
+    "start": "node index.js"
+  },
+  "dependencies": {
```

Comment:
This will also need to support PG and SQLite for us to merge it

Reply:
I named it "sql-blueprinting" for this particular reason, but I can rename it to "db-blueprinting" and add more dbs support in the future. Is it blocking if I define that it works only for SQL atm?